static/app/views/releases/list/releasesPromo.tsx

```diff
@@ -145,16 +150,12 @@ const ReleasesPromo = ({organization, project}: Props) => {
   }, [organization, project.id]);
 
   const fetchToken = async sentryAppSlug => {
```

Comment:
I wonder if we should rename this, now that it's not "fetch"ing anything and is really just a proxy to `generateToken` instead 🤔 

maybe "genreateAndSetToken" or something?

Reply:
Yeah good point, let me update

Reply:
done!